bazaarvoice/emodb
# Emo-7571 postman tests for {table} endpoint

## What Are We Doing Here?

We are extending automation testing coverage by adding postman tests. This commit is covering all kind of checks for all endpoints under /sor/1/{table} api path.

Feature coverage is additionally updated in our (for now internal) testing artefacts.

## How to Test and Verify

1. Check out this PR
2. Start emodb on local
3. Open postman
4. Import "emodb_tests.postman_collection.json" as a collection
5. Import "sor-1-_table.postman_environment.json" as environment variables
6. Execute tests in postman
7. Profit

Expected result: All tests are successfully executed on local environment

## Risk
Low
There are no changes to application code base

### Required Testing

All newly added postman tests should be executed without failures.

### Risk Summary

There are no risks to application, only tests are involved and it is about adding a new tests

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
